v0/destinations/tiktok_ads/data/TikTokTrack.json

```diff
@@ -0,0 +1,47 @@
+[
+  {
+    "destKey": "event_id",
+    "sourceKeys": "properties.eventId",
+    "required": false
+  },
+  {
+    "destKey": "timestamp",
+    "sourceKeys": "timestamp",
+    "required": false
+  },
+  {
+    "destKey": "test_event_code",
+    "sourceKeys": "properties.testEventCode",
+    "required": false
+  },
+  {
+    "destKey": "properties",
+    "sourceKeys": "properties.properties",
+    "required": false
+  },
+  {
+    "destKey": "context.ad",
+    "sourceKeys": "properties.context.ad",
+    "required": false
+  },
+  {
+    "destKey": "context.page",
+    "sourceKeys": "properties.context.page",
+    "required": false
+  },
+  {
+    "destKey": "context.user",
+    "sourceKeys": "properties.context.user",
+    "required": false
+  },
+  {
+    "destKey": "context.ip",
+    "sourceKeys": ["properties.context.ip", "context.ip"],
```

Comment:
Lets extract this from `context.ip` only

Reply:
Done